### DISCUSSION TITLE
How does photoprism do AI face detection, labeling, and geolocations?

### DISCUSSION THREAD

**Discussion by denppa** [photoprism/photoprism#3949] [General]
I am not interested in sending my data to any remote server, therefore I want to find out how photoprism is doing these three things.

These two pages seems to suggest they are done locally in the backend:
https://docs.photoprism.app/user-guide/organize/people/
https://docs.photoprism.app/user-guide/organize/labels/

I just want to make sure that the AI is indeed running on my local instance.

Secondly, I believe the geolocation feature is not enabled by default unless I explicitly set it in the settings right?

**Comment by lastzero** [comment]
We recommend that you read our privacy policy and FAQs, as these should answer any privacy-related questions you may have:
- https://docs.photoprism.app/getting-started/faq/#does-your-software-depend-on-any-external-services
- https://www.photoprism.app/privacy#section-7
- https://www.photoprism.app/kb/compliance-faq#privacy

Regarding the default settings, please note that we do not deactivate the "Places" feature by default. If you do this and later decide that you want to have the location information, your library will need to be completely re-indexed. We therefore recommend that you check all settings and config options before you start indexing your library.

**Reply by denppa** [reply]
Thank you. I was able to uncheck the undesired features. I will read through the links you so kindly provided.
